api/src/main/resources/org/openmrs/api/db/hibernate/Condition.hbm.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+	"-//Hibernate/Hibernate Mapping DTD 3.1//EN"
+	"http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.openmrs">
+
+	<class name="Condition" table="condition">
+		<id name="conditionId" type="int" column="condition_id">
+			<generator class="native">
+				<param name="sequence">condition_condition_id_seq</param>
+			</generator>
+		</id>
+
+		<property name="previousVersion" type="org.openmrs.Condition" column="previous_version" />
+		<property name="onsetDate" type="java.util.Date"
+				  column="onset_date" length="19"  />
+		<property name="condition" type="java.lang.String"
+				  not-null="true" length="50" column="condition" />
+		<property name="clinicalStatus" column="clinical_status">
+			<type name="org.hibernate.type.EnumType">
+				<param name="enumClass">org.openmrs.ConditionClinicalStatus</param>
+				<param name="type">12</param>
+			</type>
+		</property>
+		<property name="verificationStatus" column="verification_status">
+			<type name="org.hibernate.type.EnumType">
+				<param name="enumClass">org.openmrs.ConditionVerificationStatus</param>
+				<param name="type">12</param>
+			</type>
+		</property>
+		<property name="additionalDetail" type="java.lang.String"
+				  length="255" column="additional_detail" />
+		<property name="dateCreated" type="java.util.Date"
+				  column="date_created" not-null="true" length="19" />
+		<property name="voided" type="java.lang.Boolean"
```

Comment:
Can you add defaultValueBoolean="false"

Reply:
Okay. I will do that. 
Is this default value necessary because the field is not null?
@dkayiwa

Reply:
Did you see the above comment?